lib/statsd/instrument/assertions.rb

```diff
@@ -11,6 +11,10 @@ def assert_statsd_increment(metric_name, options = {}, &block)
     assert_statsd_call(:c, metric_name, options, &block)
   end
 
+  def assert_statsd_increments(metric_name, ordered_metric_options = [], &block)
```

Comment:
Not sure if adding a new method is the best choice for the API here... Maybe it's more intuitive to change the old API (and support both the old "syntax" and the new one)?